install_config/persistent_storage/persistent_storage_manila.adoc

```diff
@@ -0,0 +1,270 @@
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+{product-title} is capable of provisioning PV using the
+link:https://wiki.openstack.org/wiki/Manila[OpenStack Manila] shared file system service.
+
+It is assumed the OpenStack Manila service has been correctly set up and is
+accessible from the {product-title} cluster. Only the NFS share types can be
+provisioned.
+
+Familiarity with
+link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVs],
+link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims[persistent
+volume claims (PVCs)], and
+link:http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html[dynamic
+provisioning] is recommended.
+
+[[installation-setup]]
+== Installation and Setup
+
+The featuere is provided by an external provisioner that needs to be installed and
+configured in the {product-title} cluster.
+
+[[start-controller-and-provisioner]]
+=== Starting the External Provisioner
+
+The external provisioner service is distributed as a container image and can be run in the
+{product-title} cluster as usual. There is also RPM versions for provisioner.
+
+To allow the containers managing the API objects, the necessary role-based access control
+(RBAC) rules need to be configured by the administrator:
+
+. Create a `ServiceAccount`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: manila-provisioner-runner
+----
+
+. Create a `ClusterRole`:
++
+[source,yaml]
+----
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: manila-provisioner-role
+rules:
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["get", "list", "watch", "create", "delete"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims"]
+    verbs: ["get", "list", "watch", "update"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["storageclasses"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["list", "watch", "create", "update", "patch"]
+----
+
+. Bind the rules via `ClusterRoleBinding`:
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: manila-provisioner
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: manila-provisioner-role
+subjects:
+- kind: ServiceAccount
+  name: manila-provisioner-runner
+  namespace: default
+----
+
+. Create a new `StorageClass`:
++
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: "manila-share"
+provisioner: "externalstorage.k8s.io/manila"
+parameters:
+  type: "default"
+  zones: "nova"
+----
+
+The `type` parameter specifes the
+link:https://docs.openstack.org/manila/latest/admin/shared-file-systems-share-types.html[Manila share type]
+the provisioner will create for the volume. The `zones` parameter is a set of
```

Comment:
The 'list' is a data type and its representation is defined in the YAML syntax. Since there are two ways to express it I would rather leave it up to the user to choose.